Cache/IO.lean

```diff
@@ -30,10 +30,21 @@ def Nat.toHexDigits (n : Nat) : Nat → (res : String := "") → String
 def UInt64.asLTar (n : UInt64) : String :=
   s!"{Nat.toHexDigits n.toNat 8}.ltar"
 
+open Lean
+
 namespace Cache.IO
 
 open System (FilePath)
 
+/--
+Read the search path from `LEAN_PATH` and drop all trailing `/.lake/build/lib/`
+as the `.lean`-files are located outside the `.lake/` folders.
```

Comment:
If you're looking for `.lean` files, you should be using `LEAN_SRC_PATH` not `LEAN_PATH`.

Reply:
So call `initSrcSearchPath` or similar instead.

Reply:
Thanks you're right. I confused myself by importing `Lean.Util.Path` instead of `Lean.Util.Paths`